engine/src/main/java/nl/inl/blacklab/codec/BlackLab40PostingsWriter.java

```diff
@@ -501,16 +502,22 @@ private void writeTokensInDoc(IndexOutput outTokensIndexFile, IndexOutput outTok
                     for (int token: tokensInDoc) {
                         outTokensFile.writeShort((short) token);
                     }
-                    break; 
-                case INT: 
+                    break;
+                case THREE_BYTES:
+                    for (int token : tokensInDoc) {
+                        outTokensFile.writeByte((byte)(token >> 16));
+                        outTokensFile.writeByte((byte)(token >>  8));
+                        outTokensFile.writeByte((byte) token);
+                    }
```

Comment:
Why not:
```java
                        outTokensFile.writeByte((byte)(token >>  16));
                        outTokensFile.writeShort((short) token);
```

Reply:
No specific reason, this is just the implementation of internal readInt() with the lowest byte removed

Reply:
Right, didn't think about that, that makes sense. I guess it doesn't matter performance-wise then.